docs/Reference/RPC/transitionDeviceRPC.md

```diff
@@ -0,0 +1,51 @@
+--8<-- "References/abbreviations.md"
+
+If an Intel vPro® Platform has been previously activated, either in the BIOS or with another management solution or tool, it can be brought under Open AMT Cloud Toolkit control with the RPC-Go application. 
+
+Additionally, use the following instructions to transition from a previously established toolkit stack to a fresh installation on a new development system.
+
+!!! Note "Note - Passwords used in Open AMT Cloud Toolkit"
+    Use the following instructions to transition devices to either ACM or CCM profiles. You will need the AMT password.
+    
+    Open AMT Cloud Toolkit increases security with multiple passwords. Find an explanation of toolkit passwords in [Reference -> Architecture Overview](../architectureOverview.md#Passwords).
+
+**To transition the activated device:**
+
+1. Check the activation status with `amtinfo` command:
+
+    === "Linux"
+        ``` bash
+        sudo ./rpc amtinfo
+        ```
+    === "Windows"
+        ```
+        .\rpc amtinfo
+        ```        
+    === "Docker (On Linux Host Only)"
+        ``` bash
+        sudo docker run --rm -it --device=/dev/mei0 rpc-go:latest amtinfo
+        ```
+    
+    The **control mode** indicates the managed device's state:
+
+    - pre-provisioning or deactivated
+    - activated in client control mode (CCM)
+    - activated in admin control mode (ACM)
+
+2. Run the rpc-go application with the `activate` command and the `-password` flag:
+
+    === "Linux"
+        ``` bash
+        sudo ./rpc activate -u wss://[RPS-IPAddress-or-FQDN]/activate -n -profile [profileName] -password [AMTPassword]
```

Comment:
Probably use [Development-IP-Address] instead of [RPS-IPAddress-or-FQDN], customers might get confused if both of those are different